fastlane_bot/helpers/poolandtokens.py

```diff
@@ -476,6 +485,67 @@ def decimal_converter(idx):
                 "descr": self.descr,
                 "params": self._params,
             }
+
+            strategy_typed_args += [typed_args]
+
+        #### MODIFICATION LOGIC HERE >>>>>
+        # Only overlapping strategies are selected for modification
+        if len(strategy_typed_args) == 2:
```

Comment:
Unrelated to the changes in this PR, but each one of the two attempts to "try / except decimal.InvalidOperation" is completely redundant, because if the code prior to this attempt does not throw any exception, then so will the code inside this attempt.